controller/handler/patient.js

```diff
@@ -81,14 +81,17 @@ function patientView (request, reply) {
                 patient: processPatient(currentPatient),
                 trial: currentTrial,
                 surveys: surveyInstances.map((surveyInstance) => {
-                    surveyInstance.startTime = moment(surveyInstance.startTime).format('MM-DD-YYYY');
-                    surveyInstance.endTime = moment(surveyInstance.endTime).format('MM-DD-YYYY');
+                    surveyInstance.startTime = moment(new Date(surveyInstance.startTime)
```

Comment:
The "new Date()" is used to convert the date to ISO string format. As there will not be support for strings in Moment going forward.
I converted the date to UTC because I am doing the same for all the dates I get and this will ensure sync.